documentation/chatmeleon-swagger.yaml

```diff
@@ -248,14 +399,18 @@ components:
           type: string
           format: date-time
           description: The timestamp when the message was created.
+        seenIds:
+          type: array
+          items:
+            type: string
+            pattern: "^[0-9a-f]{24}$"
```

Comment:
What is this field? Maybe add description?

Reply:
Okay, ill add one